app/elements/pw-shell.html

```diff
@@ -161,11 +158,11 @@
       }
     </style>
 
-    <app-location route="{{_sectionRoute}}"></app-location>
-    <app-route route="{{_sectionRoute}}" pattern="/:version/:section" data="{{_sectionData}}" tail="{{_subsectionRoute}}"></app-route>
-    <app-route route="{{_sectionRoute}}" pattern="/search/" query-params="{{_searchParams}}"></app-route>
+    <app-location route="{{_route}}" path="{{path}}"></app-location>
```

Comment:
`<app-location>` internally keeps `route.path` the same value as `path`, but in our case we can just use `path` directly and not rely on `this.set`. (We still need `route` for the `<app-routes>` below though.)